### ISSUE TITLE
Failed to build base-notebook, /bin/sh: 1: fix-permissions: not found

### ISSUE THREAD

**Issue by yttty** [jupyter/docker-stacks#696]
**What docker image you are using?**

`jupyter/base-notebook`

**What complete docker command do you run to launch the container (omitting sensitive values)?**

`docker build -t base-notebook .`

**What actually happens?**

Step 9/27 : ADD fix-permissions /usr/local/bin/fix-permissions
 ---> Using cache
 ---> dbbd6b5f9cca

Step 10/27 : RUN groupadd wheel -g 11 &&     echo "auth required pam_wheel.so use_uid" >> /etc/pam.d/su &&     useradd -m -s /bin/bash -N -u $NB_UID $NB_USER &&     mkdir -p $CONDA_DIR &&     chown $NB_USER:$NB_GID $CONDA_DIR &&     chmod g+w /etc/passwd &&     fix-permissions $HOME &&     fix-permissions $CONDA_DIR
 ---> Running in 1b1a8dfd838c
/bin/sh: 1: fix-permissions: not found

**Comment by parente** [comment]
Have you cloned the entire git repo? Please check if there's a fix-permissions script in the folder where you're running `docker build`.

**Comment by yttty** [comment]
Thanks. There is such a script. 

```
~/WORKSPACE/docker-stacks/base-notebook (master)
$ git status
On branch master
Your branch is up to date with 'origin/master'.

nothing to commit, working tree clean
```

I tried to change that line in Dockerfile to `COPY fix-permissions /usr/local/bin/` but the problem still exists.

**Comment by yttty** [comment]
Solved it by changing EOF from CRLF to LF.

**Comment by parente** [comment]
Line endings problem should be resolved by PR #704. Closing this out.
